### PR TITLE
Update packages

### PR DESCRIPTION
**Pull request by tmccoy529**
Updated the pets working cats informational div to take a value from the windows variables instead of being hardcoded.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -30,9 +30,9 @@
     <script>
       window.pets = {
         informationAbout:
-          "If you're interested in adopting an animal, please have its Animal ID: {animalId} handy. Before calling or visiting our shelter, please review our page <a href='https://beta.baltimorecountymd.gov/departments/animalservices/how-to-adopt.html'>how to adopt</a>.",
+          `<h3>Information About Working Cat Adoption</h3><p>Adopt one or several of our working cats shown below and help save lives today! Learn more about our<a href='https://www.baltimorecountymd.gov/departments/animalservices/working-cat-program.html'>Working Cat Program</a>.</p>`,
```
Comment (srutherfordbcg) [inline public/index.html:33]: We need a space between "Learn more about our" and the link 😄

Comment (tmccoy529) [reply public/index.html:33]: Hahaha ugh, can you make that change for me?

File: src/components/PetsList.jsx [hunk 2]
```diff
@@ -5,20 +5,29 @@ import React from "react";
...
-    apiEndpoint={`${getValue("apiRoot")}`}
-    renderItem={(props) => <PetCard {...props} />}
-    includeInputFilter={true}
-    includeClearButton={true}
-    searchCategory="Pets"
-    inputFilterPlaceholder="Begin typing to filter the results..."
-  />
+  <div>
+    {workingCats === "true" ? (
+      <div dangerouslySetInnerHTML={{ __html: informationAbout }}></div>
```
Comment (srutherfordbcg) [inline src/components/PetsList.jsx:18]: I guess there is no safelySetInnerHTML 😄

**Review by srutherfordbcg — APPROVED** [review]

File: public/index.html (hunk 1 above)
Comment (srutherfordbcg) [reply public/index.html]: done!

**Review by sgrg1 — APPROVED** [review]

File: src/components/PetsList.jsx (hunk 2 above)
Comment (ascire) [reply src/components/PetsList.jsx:18]: I've seen this done in configuration and APIs before: the self-documenting "are you sure you know what you're doing!?" method or function.

**Pull request merged**